src/config/dynamic-theme-fixes.config

```diff
@@ -15362,6 +15362,19 @@ a#commonLogo {
 
 ================================
 
+novinky.cz
+seznamzpravy.cz
+
+CSS
+.ribbon, .szn-input-with-suggest-list {
+    background-color: var(--darkreader-neutral-background) !important;
+}
+.szn-input-with-suggest-list {
+    border-color: #494949 !important;
```

Comment:
Is there any particular reason for this specific color? Would `${...}` color adjustment work better?